.gitlab-ci.yml

```diff
@@ -124,11 +125,21 @@ test/clang-python3:6.0:
 test/rocm-python3:latest:
   extends: .test_template
   needs: ["rocm-python3:latest:build"]
+  script:
+    - git clone --depth=1 --recursive https://github.com/espressomd/espresso
+    - cd espresso
+    - maintainer/CI/build_cmake.sh
+    - cd build
+    - ctest -L gpu -LE long -E '^(tutorial|sample)_' --output-on-failure
   variables:
     make_check: "false"
     myconfig: "maxset"
     HIP_PLATFORM: "hcc"
     HCC_AMDGPU_TARGET: "gfx900"
+  tags:
+    - docker
+    - linux
+    - amdgpu
```

Comment:
Please set a 1h timeout. That‘s what we have in the main repo and running the tests does take a bit of time. The current 30m timeout was intended for compiling only.

Reply:
I manually ran the test on coyote10  before opening the PR and it only took 4 min longer.

Reply:
Did you take into account that CI (but not manual testing) sets CPU limits and the machine was probably idle when you were testing?

Reply:
The CPU limit should not have too much of an impact since GPU python tests cannot run in parallel. But you're right, the number of other jobs running on the machine can have an impact. The timeout was increased to 1h.